server/src/main/scala/org/ensime/server/Server.scala

```diff
@@ -115,13 +117,23 @@ object Server extends AkkaBackCompat {
   Slf4jSetup.init()
 
   val log = LoggerFactory.getLogger("Server")
+  val XDG_CONFIG_HOME = " ~/.config/ensime-server"
```

Comment:
@hardmettle we have three file to check for with priority order

1. project-root/.ensime-server.conf
2. $XDG_CONFIG_HOME(or ~/.config/)/ensime/server.conf -- this is in compliance with xdg directory specification we can deviate if we choose to.
3. ~/.ensime-server.conf